### ISSUE TITLE
When I click on the test summary line, the runner quits instead of showing the tests

### ISSUE THREAD

**Issue by NameOfTheDragon** [xunit/devices.xunit#50]
I'm running this on a Raspberry Pi 3, with Windows 10 IoT Core 10.0.14393.576 "Anniversary update".

My unit test app builds, deploys and runs as expected. I can click "Run Everything" and all the tests run (some fail, this is expected).

However, if I click on the summary bar where it tells me how many tests there are, the runner app simply quits instead of showing the test details.

**Comment by onovotny** [comment]
@NameOfTheDragon Does this repro on "regular" Desktop windows or Mobile? Is there an IoT Core emulator? Not sure how I can repro this otherwise.

Happy to take a PR or otherwise more info?

**Comment by NameOfTheDragon** [comment]
@onovotny No, this doesn't reproduce on desktop Windows 10. However, since this is a device runner, I wouldn't have thought that was really the prime use case. It is 100% reproducible on my Raspberry pi, though, even with an new test runner. The minimum code required to reproduce is here: https://bitbucket.org/tigra-astronomy/xunit.pirunner

I'm not sure that I'm really set up to investigate this myself but I will try to have a go at it over the next few days. Alternatively, if it would help, I can arrange to publish my Raspberry Pi through the firewall so you can debug it remotely. It wouldn't be ideal but it would at least make an investigation possible, and probably more realistic that me getting up to speed on building and debugging xunit.

**Comment by NameOfTheDragon** [comment]
I was able to build and deploy test.xunit.runner.uwp to my Raspberry Pi and when clicking on a list item, I get the following unhandled exception:
![image](https://cloud.githubusercontent.com/assets/1268490/21472315/aa866e3c-cacb-11e6-82c2-6ecab93367a5.png)

**Comment by NameOfTheDragon** [comment]
OK, sure enough, removing that `SearchBox` from `AssemblyTestListPage.xaml` fixes the problem and allows me to expand the list items. I don't think there's much point in me submitting a PR with just that deletion, ideally we would need to find a replacement for the `SearchBox` control but I'm not sure how to mitigate this at the moment, so that's as far as I think I can take it. I think you have your 'smoking gun' though.

**Comment by onovotny** [comment]
Thanks, let me check with a few others about this.

Sent from my Windows 10 phone

**Comment by NameOfTheDragon** [comment]
There’s no rush… enjoy Christmas! ☺

Best regards,
Tim Long

**Comment by onovotny** [comment]
I tried to fix this by using the `AutoSuggestComboBox` as recommended. Can you try the CI feed and let me know if it works for you on IoT?

The CI feed is `https://www.myget.org/F/xunit-xamarin/api/v3/index.json` and the fix is in 2.2.0-rc2-build0002 or later.

**Comment by NameOfTheDragon** [comment]
Thanks Oren, I will do my best. My Raspberry Pi system is not set up at the moment but I should be able to try something soon.
--Tim
